baseframe/forms/fields.py

```diff
@@ -725,3 +726,24 @@ def process_formdata(self, valuelist):
     def pre_validate(self, form):
         if self.data is _invalid_marker:
             raise ValueError(self.gettext('Not a valid choice'))
+
+
+class JSONField(wtforms.TextAreaField):
+    def _value(self):
+        if self.raw_data:
+            return self.raw_data[0]
+        elif self.data:
+            return json.dumps(self.data, ensure_ascii=False)
```

Comment:
Are you sure you want `ensure_ascii=False`? https://stackoverflow.com/a/40421631/78903